templates/mount.sh.tmpl

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
One last change, I tested this script and unfortunately it doesn't work with nvme volumes like on the t3a.small because of the way AWS handles device naming. I made some modifications and tested the following provisioner block and it does work, and does ensure the volume stays attached when rebooted:
```
provisioner "remote-exec" {
    inline = [
      "EBS_DEVICE_NAME=$(lsblk | grep ${var.EBS_MEDIA_VOLUME_SIZE}G | awk '{print $1}')",
      "sudo mkfs -t xfs /dev/$${EBS_DEVICE_NAME}",
      "EBS_DEVICE_UUID=$(sudo blkid | grep $${EBS_DEVICE_NAME} | awk -F'\"' '{print $2}')",
      "sudo echo -e \"UUID=$${EBS_DEVICE_UUID} /home/ec2-user/jellyfin/media  xfs  defaults,nofail  0  2\" | sudo tee -a /etc/fstab",
      "sudo mount -a",
      "sudo chown -R ec2-user /home/ec2-user/jellyfin/media"
    ]
  }
```
This is a bit awkward because it has to detect the disk from the unique disk size. Unfortunately AWS doesn't really provide any straightforward mapping, and reserves the right to change device names based on the hardware it's running on, so this might be the best solution. (See https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/device_naming.html)

Also, remote-exec would be better here since passing this as a file to user_data doesn't seem to run this when the instance is created.